[BUGFIX] Correction d'affichage de l'ampoule sur la page "Réponses et tutos" (PIX-1074)

alexandrecoin opened the pull request:
## :unicorn: Problème
Sur l'écran "Réponses et tutos", la taille de l'ampoule "Pour réussir la prochaine fois"change en fonction de la hauteur du texte descriptif. Elle est des fois illisible.

<img width="349" alt="Capture d’écran 2020-08-03 à 11 53 40" src="https://user-images.githubusercontent.com/36371437/89170883-988b0800-d580-11ea-9317-4e7b511a2c06.png">


## :robot: Solution
Correction du display flex du container et ajout de padding afin d'améliorer le centrage en version mobile

<img width="908" alt="Capture d’écran 2020-08-03 à 11 51 16" src="https://user-images.githubusercontent.com/36371437/89171029-d4be6880-d580-11ea-8027-e877fdf2849f.png">

<img width="369" alt="Capture d’écran 2020-08-03 à 11 51 09" src="https://user-images.githubusercontent.com/36371437/89171051-dbe57680-d580-11ea-9440-55e24b7e3394.png">


## :100: Pour tester
Réaliser un parcours et cliquer sur Réponses et tutos dans une page de mi-parcours.

alexandrecoin committed:
Fix lightbulb display on tutorial page

pix-service commented:
I'm deploying this PR to these urls:

- App: https://app-pr1725.review.pix.fr
- Orga: https://orga-pr1725.review.pix.fr
- Certif: https://certif-pr1725.review.pix.fr
- Admin: https://admin-pr1725.review.pix.fr
- API: https://api-pr1725.review.pix.fr/api/

 Please check it out!

jonasgrilleres reviewed (approved):
Juste un petit commentaire sur le nom
de ton message de commit : les titres de commits doivent pouvoir compléter la phrase "if applied, this commit will ..." (cf. [https://github.com/1024pix/pix/blob/dev/docs/CONTRIBUTING.md](CONTRIBUTING.md)). Ça donnerait plutôt "Fix lightbulb..."

celineung commented:
J'ai un petit soucis d'alignement en mobile:
<img width="352" alt="Screenshot 2020-08-03 at 14 24 06" src="https://user-images.githubusercontent.com/11294487/89181998-0d683d00-d595-11ea-83ba-712c2f8bb162.png">

Peut-être supprimer le padding dans `.tutorial-panel__hint-title span` ?

alexandrecoin commented:
> J'ai un petit soucis d'alignement en mobile:
> <img alt="Screenshot 2020-08-03 at 14 24 06" width="352" src="https://user-images.githubusercontent.com/11294487/89181998-0d683d00-d595-11ea-83ba-712c2f8bb162.png">
> 
> Peut-être supprimer le padding dans `.tutorial-panel__hint-title span` ?

Le problème semble effectivement venir du padding. 
Le supprimer n'est toutefois pas la solution puisque cela engendrerait un nouveau problème de barre grise collée au texte pour toute résolution supérieure à celle pour laquelle le problème se pose.
Et cela ne peut être résolu par un @includes puisque la largeur maximale du format mobile permet à la fois de reproduire le décalage et de le fixer ( width > 356px) :/

HEYGUL committed:
Fix padding issues on small screen.

I applied solution #2 of this article https://css-tricks.com/multi-line-padded-text.

HEYGUL commented:
> > J'ai un petit soucis d'alignement en mobile:
> > <img alt="Screenshot 2020-08-03 at 14 24 06" width="352" src="https://user-images.githubusercontent.com/11294487/89181998-0d683d00-d595-11ea-83ba-712c2f8bb162.png">
> > Peut-être supprimer le padding dans `.tutorial-panel__hint-title span` ?
> 
> Le problème semble effectivement venir du padding.
> Le supprimer n'est toutefois pas la solution puisque cela engendrerait un nouveau problème de barre grise collée au texte pour toute résolution supérieure à celle pour laquelle le problème se pose.
> Et cela ne peut être résolu par un @includes puisque la largeur maximale du format mobile permet à la fois de reproduire le décalage et de le fixer ( width > 356px) :/

Je viens de pousser un commit qui corrige le soucis. Je me suis basé sur la deuxième solution expliquée dans cet article : https://css-tricks.com/multi-line-padded-text/.

frinyvonnick commented:
En testant sur desktop, j'ai eu cette affichage là :

![image](https://user-images.githubusercontent.com/13099512/89266187-e31a8c00-d635-11ea-870e-afa2d35c6bbe.png)

MelanieMEB reviewed (changes requested):
Le problème d'ampoule qui change de taille en fonction du texte est toujours présent.
Un `min-width: 100px;` sur `.tutorial-panel__hint-picto` semble aider au problème.

HEYGUL commented:
> Le problème d'ampoule qui change de taille en fonction du texte est toujours présent.
> Un `min-width: 100px;` sur `.tutorial-panel__hint-picto` semble aider au problème.

Effectivement, mais pas sur Firefox :/
